sys/net/gnrc/routing/ipv6_auto_subnets/gnrc_ipv6_auto_subnets.c

```diff
@@ -302,8 +304,12 @@ static void _configure_subnets(uint8_t subnets, uint8_t start_idx, gnrc_netif_t
         _remove_old_prefix(downstream, &new_prefix, new_prefix_len, &ext_opts);
```

Comment:
This function is called on every received RA, so quite often. Can we also `continue` or even `return` if the prefix did not change, i.e. this function returns true?

Reply:
This is already happening in [`gnrc_ipv6_auto_subnets.c:L228`](https://github.com/RIOT-OS/RIOT/blob/5785856fe711bbee8acda5a6c5780c9b485476bd/sys/net/gnrc/routing/ipv6_auto_subnets/gnrc_ipv6_auto_subnets.c#L228)